### ISSUE TITLE
Does EaselJS using WebGL premultiply twice?

### ISSUE THREAD

**Issue by Rigwarl** [CreateJS/EaselJS#901]
Hello, when I'm using `StageGL` with`premultiply: true` to avoid visual halo effects my textures become A LOT darker.

As I can see EaselJS uses this in [`canvas.getContext("webgl", options)`](http://www.createjs.com/docs/easeljs-stagegl/files/easeljs_display_StageGL.js.html#l1454)
and in [`gl.pixelStorei(gl.UNPACK_PREMULTIPLY_ALPHA_WEBGL, this._premultiply)`](http://www.createjs.com/docs/easeljs-stagegl/files/easeljs_display_StageGL.js.html#l914)

When I comment last one out premultiply works fine (no halo effects) and textures become just a bit darker. It seems like EaselJS premultiplying alpha twice.

I am very new to WebGL. Sorry if I'm asking something obvious.

**Comment by DavidHGillen** [comment]
We've done some deep diving on this topic and there's definitely room for improvement in the current build. If you wanted to try https://github.com/DavidHGillen/EaselJS/tree/BlendModes to see if we've resolved all your issues with premultiplied alpha I wouldn't have any complaints. The option has been removed from the constructor but I believe we've corrected the behavior to what's expected for most if not all content, thus removing the need for it. Excuse any untidiness, the code is still under development.
